profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/EtwEventsManager.h

```diff
@@ -110,6 +110,7 @@ class EtwEventsManager :
 
     // responsible for receiving ETW events from the Windows Agent
     std::unique_ptr<EtwEventsHandler> _eventsHandler;
+    std::string _agentEndpoint;
```

Comment:
`_agentEndpoint` is for the replay scenario only ?
if yes, maybe make it explicit can help understand that's not a normal case (in prod)